CVLByExample/Reentrancy/certora/conf/ReentrancyVulnerableBankBadFix.conf

```diff
@@ -0,0 +1,13 @@
+{
+    "files": [
+        "contracts/VulnerableBankBadFix.sol"
+    ],
+    "loop_iter": "3",
+    "optimistic_loop": true,
+    "verify": "VulnerableBankBadFix:certora/spec/Reentrancy.spec",
+    "prover_args": [
+         "-ignoreViewFunctions"
+    ],
+    "prover_version":"shelly/curve",
```

Comment:
Is this the right branch? Will it always stay?

Reply:
we will use a dedicated version until it is in master